client/upe-onboarding-wizard/upe-preview-methods-selector/__tests__/setup-complete-task.test.js

```diff
@@ -1,52 +1,148 @@
-import React from 'react';
 import { render, screen } from '@testing-library/react';
 import WizardTaskContext from '../../wizard/task/context';
 import SetupComplete from '../setup-complete-task';
 import WizardContext from '../../wizard/wrapper/context';
+import { useEnabledPaymentMethodIds } from 'wcstripe/data';
+
+jest.mock( 'wcstripe/data', () => ( {
+	useEnabledPaymentMethodIds: jest.fn(),
+} ) );
 
 describe( 'SetupComplete', () => {
-	const renderHelper = ( setCompletedMock ) => {
-		if ( ! setCompletedMock ) {
-			setCompletedMock = jest.fn();
-		}
+	beforeEach( () => {
+		useEnabledPaymentMethodIds.mockReturnValue( [
+			[ 'card', 'giropay', 'sofort' ],
+			() => null,
+		] );
+	} );
```

Comment:
We may [clear the mocks](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1887/files?file-filters%5B%5D=.js&hide-deleted-files=true#r711949376) in this file too.

Reply:
The `beforeEach` is already resetting the implementation of the one mocked function in the file - what would `clearAllMocks()` provide in this case?

Reply:
It does not provide much in this specific case. I had previously experienced cases where not resetting mocks in one test made some other tests fail indefinitely. It generally happens when you have different mocks in different `it` blocks. In that cases clearing everything in `afterEach` provides a safe test environment and helps avoid all those indefinite random test failures.